students/ericstreit/.gitignore

```diff
@@ -0,0 +1,106 @@
+.DS_Store
```

Comment:
you don't most (any?) of these, 'cuase they are in the main .gitignore.

but it won't hurt, either.